File: SWB4/Client/OpenOffice/OfficeComunicationInterfaces/src/org/semanticwb/office/interfaces/FlowContentInformation.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.semanticwb.office.interfaces;

/**
 *
 * @author victor.lorenzana
 */
public class FlowContentInformation {
    public ResourceInfo resourceInfo;
    public String id;
    public String title;
    public String step;
    public int status;

    @Override
    public String toString()
    {
        return title.toString();
    }


}

File: SWB4/Client/Microsoft Office/INFOTEC WebBuilder 4/WBOffice4/Interfaces/FlowContentInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WBOffice4.Interfaces
{
    public class FlowContentInformation {
    public ResourceInfo resourceInfo;
    public String id;
    public String title;
    public String step;
    public int status;

    
    public override String ToString()
    {
        return title.ToString();
    }


}
}
